beast/plotting/plot_cmd_with_fits.py

```diff
@@ -205,26 +203,22 @@ def plot_cmd_with_fits(
     )
     parser.add_argument(
         "--log_param",
-        action="store_true",
-        help="choose whether to take the log of `param` for assigning color",
-    )
-    parser.add_argument(
-        "--savefig",
-        action="store_true",
-        help="choose whether to save the figure, or show it",
+        type=int,
+        default=0,
+        help="If you would like to take the log of `param` for assigning color, set this to 1 (True)",
```

Comment:
Why change this to an int when it is behaving as as a boolean?

Reply:
Having the default be False should be action of log_param is not set with store_true, ,so not sure why this change is even needed.

Reply:
There were some other places we'd changed true/false keywords to ints.  But maybe that was where turning on the keyword made the input False, which was simpler to do as an int (and then changed to boolean).  Happy to change this one back.

Reply:
Great.  We should use this argparse capability where we can - less code and easier to understand what is happening including the argparse commandline help.